com.unity.render-pipelines.universal/Editor/ShaderGraph/Includes/UnlitPass.hlsl

```diff
@@ -28,5 +28,9 @@ half4 frag(PackedVaryings packedInput) : SV_TARGET
     surfaceDescription.BaseColor *= surfaceDescription.Alpha;
 #endif
 
+#if defined(_DBUFFER)
+    ApplyDecalToBaseColor(unpacked.positionCS, surfaceDescription.BaseColor);
```

Comment:
Just a remark: We don't support decal on unlit in HDRP, I guess you have get the request from your artist?

Reply:
Yes